code/game/objects/items/devices/paicard.dm

```diff
@@ -164,6 +164,15 @@
 
 /obj/item/paicard/proc/alertUpdate()
 	audible_message("<span class='info'>[src] flashes a message across its screen, \"Additional personalities available for download.\"</span>", "<span class='notice'>[src] vibrates with an alert.</span>")
+	cut_overlays()
+	add_overlay("pai-alert")
+	addtimer(CALLBACK(src, .proc/clearalert), 2 MINUTES)
+
+/obj/item/paicard/proc/clearalert()

```

Comment:
add logic to `/datum/controller/subsystem/pai/Topic` to automatically call clearalert() on all pAI cards if there are no remaining candidates

Reply:
I'll replace the timed clear with this since it's more efficient

Gave it my best shot and it's not working at the moment since I'm kinda winging it, could use some advice